src/api/AccountApi.ts

```diff
@@ -281,6 +281,26 @@ export class AccountApi extends BaseApi {
             this.api.broadcastApi.broadcastWithCallback(credentials.keyPair, [operation])));
     }
 
+    /**
+     * Create a register new account operation.
```

Comment:
Typo?

Reply:
no, wanted to avoid `Create a create new account operation.`

Reply:
OK